utils/win32/CMakeSettings.json

```diff
@@ -0,0 +1,95 @@
+﻿{
+  "configurations": [
+    {
+      "name": "x64-Debug-static",
+      "generator": "Ninja",
+      "configurationType": "Debug",
+      "inheritEnvironments": [ "msvc_x64_x64" ],
+      "buildRoot": "${projectDir}\\build\\${name}",
+      "installRoot": "${projectDir}\\build\\${name}-install",
+      "cmakeCommandArgs": "",
+      "buildCommandArgs": "",
+      "ctestCommandArgs": "",
+      "variables": [
+        {
+          "name": "VCPKG_TARGET_TRIPLET",
+          "value": "x64-windows-static",
+          "type": "STRING"
+        },
+        {
+          "name": "OPTION_BUILD_WEBSITE_TOOLS",
+          "value": "False",
+          "type": "BOOL"
+        },
+        {
+          "name": "OPTION_BUILD_WINSTATIC",
+          "value": "True",
+          "type": "BOOL"
+        },
+        {
+          "name": "OPTION_ASAN",
+          "value": "False",
+          "type": "BOOL"
+        },
+        {
+          "name": "OPTION_BUILD_CODECHECK",
+          "value": "False",
+          "type": "BOOL"
+        },
+        {
+          "name": "OPTION_USE_GLBINDING",
+          "value": "True",
```

Comment:
this setting is mutually exclusive with OPTION_BUILD_WINSTATIC as the static build works only with glew as glbinding doesn't provide a static library as far as I know at least for MSYS environment

Reply:
This is only true for MSYS. With vcpkg glbinding works fine, but I got errors using glew.

Reply:
Ok but do you really get a static linked exe without the need of the glbinbding dll?

Reply:
No glbinding dll needed. However, the debug build requires a c runtime dll, which is only installed with VS and not with the normal vc_redistributable. This should be omitted with /MTd. I will check, whether we can remove this dependency.